Singletons/Settings.gd

```diff
@@ -80,14 +85,21 @@ func get_bool_setting(setting: String) -> bool:
 	return value
 
 
-func set_setting(setting: String, value: Variant):
+func set_setting(value: Variant, setting: String):
```

Comment:
Can you revert back to the old order of arguments?

Value first and setting second doesn't make sense.
For example, ProjectSettings.set_setting():
https://docs.godotengine.org/en/stable/classes/class_projectsettings.html#class-projectsettings-method-set-setting

Reply:
I need this order to use it as a parameter for `signal.connect(Settings.set_setting.bind("setting_name"))` clause. Otherwise, I need to create a function with just the same arguments but in different order.

Reply:
`signal.connect(Settings.set_setting.bind("setting_name"))` <- this line is too complex anyway. I changed it to separate callback function and this way arg order can be good as well. Added a commit.